src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -7,13 +7,23 @@
 using System;
 using System.IO;
 using System.Runtime.Serialization;
+using System.Security;
```

Comment:
This shouldn't be necessary... (as a sub-namespace of System.Security it's implicitly using'd)